packages/components/src/instructionList/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react';
+
+export interface Instruction {
+  icon: ReactNode;
+  text: ReactNode;
+  help?: {
+    title?: ReactNode | null;
```

Comment:
`null` is a valid `ReactNode`.
```suggestion
    title?: ReactNode | null;
```

Reply:
Hmm, I don't want that. `icon` and `text` shouldn't allow null :thinking: